### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by anshul-srivastava**

### CODE REVIEW

**Comment by Kaushik512** [comment]
@revinjacob can you please review and merge this

File: client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js [hunk 1]
```diff
@@ -156,14 +162,25 @@
...
-							alert('Please select a node or blueprint');
+
+						if (!taskJSON.nodeIds.length && !taskJSON.blueprintIds.length && !taskJSON.role ) {
+							alert('Please select a node or blueprint or role');
 							return false;
 						}
 						if (taskJSON.nodeIds.length && taskJSON.blueprintIds.length) {
-							alert('Please choose either nodes or blueprints, not both');
+							alert('Please choose either nodes or blueprints or role, not all');
 							return false;
```
Comment (revinjacob) [inline client/cat3/src/partials/sections/dashboard/workzone/orchestration/popups/newTaskCtrl.js]: Condition need to be updated if blueprints and roles are both selected

**Comment by revinjacob** [comment]
@anshul-srivastava, This PR has conflicts. Please resolve and update.

**Comment by revinjacob** [comment]
Code changes look fine, merging

**Pull request merged**